README.md

```diff
@@ -2,7 +2,7 @@
      that smart and dearly wants an empty line before a heading to be able to
      display it as such, e.g. website) -->
 
-# Design patterns implemented in Java
+# Design patterns implemented in JAVA
```

Comment:
I captured that from my book shelve "D LOL

![IMG20201021161429](https://user-images.githubusercontent.com/21141301/96732458-a3ec1380-13b8-11eb-9214-452a3cfaac7c.jpg)